src/auth/auth-manager.js

```diff
@@ -0,0 +1,168 @@
+const jwt = require('jsonwebtoken');
+const User = require('../users/user-models').User;
+const Session = require('./auth-models').Session;
+
+/**
+ * Callback for sending the response to the client.
+ *
+ * @function loginResponse
+ * @param {(string|Object)} err - The error.
```

Comment:
I was passing a `string` as a parameter in a few `next` calls. Switching to an error `object` is easy change.